### PR TITLE
Css tuning

### PR DESCRIPTION
**Pull request by SamOlivas**
Optional PR
- Cart icon added
- spacing of login adjusted

### CODE REVIEW

**Review by roboMoose — APPROVED** [review]

File: src/components/CartHeader.js [hunk 1]
```diff
@@ -6,8 +6,11 @@ import { Link } from 'react-router-dom';
 export function CartHeader({ cart }) {
   const cartItems = cart ? cart.length : 0;
   return (
-    <Link to="/myCart">
-      <button id="cart-button">Cart:{cartItems}</button>
-    </Link>
+    <div className='cart-header'>
+      <Link to='/myCart'>
+        <img id='cart-icon' src='/img/cart.png'></img>
```
Comment (nicknapoli82) [inline src/components/CartHeader.js:11]: Is `/myCart` a valid link?
I forget

Comment (roboMoose) [reply src/components/CartHeader.js:11]: shoot - I didnt mean to merge

File: src/components/SingleProduct.js [hunk 2]
```diff
@@ -29,7 +29,7 @@ function SingleProduct({ product }) {
               : ''}
           </p>
           <p>
-            Cost: {product.cost} Golds
+            Cost: {product.cost} Gold
             <br />
```
Comment (nicknapoli82) [inline src/components/SingleProduct.js:33]: I intentionally misspelled golds here, but I'll take the correctness.

File: src/components/UserProfile.js [hunk 3]
```diff
@@ -12,28 +12,32 @@ class UserProfile extends React.Component {
...
+          <div id='userInfo'>
+            First Name: {user.firstName}<br/>
+            Last Name: {user.lastName}<br/>
+            Class: {user.class}<br/>
+            Email:{user.email}<br/>
+          </div>
+          <Link to="/edit-user">Edit Profile Info</Link>
+          <div className='admin-tools'>
+            {user.isAdmin ? <a href="/edit-user">Edit Current Users</a> : ''}
+          </div>
```
Comment (nicknapoli82) [inline src/components/UserProfile.js:28]: Links not being specified as Link will hard reload our application. This should not be a major problem, but is that what we want to do here?

File: src/components/CartHeader.js (hunk 1 above)
Comment (SamOlivas) [reply src/components/CartHeader.js:11]: Yup yup, /myCart is valid. Or at least working on my machine

File: src/components/SingleProduct.js (hunk 2 above)
Comment (SamOlivas) [reply src/components/SingleProduct.js:33]: Haha, I am a fan of the whimsical spellings

File: src/components/UserProfile.js (hunk 3 above)
Comment (SamOlivas) [reply src/components/UserProfile.js:28]: Oh..lemme double check again locally

Comment (SamOlivas) [reply src/components/UserProfile.js:28]: Items are persisting in cart while using Link>
Totally optional to push to Herkou, I'm nervous about breaking things before presentation too haha

**Pull request merged**